docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.4'
+
+x-common-config: &common_config
+  volumes:
+    - home:/home
+  environment:
+    - SQUAD_CELERY_BROKER_URL=amqp://rabbitmq
+    - XDG_DATA_HOME=/home
+
+volumes:
+  home:
+
+services:
+  rabbitmq:
+    image: rabbitmq
+
+  dbmigrate:
+    <<: *common_config
+    image: squad
+    command: squad-admin migrate
+    deploy:
```

Comment:
I'm getting a warning on this line:

    WARNING: Some services (dbmigrate) use the 'deploy' key, which will be ignored. Compose does not support 'deploy' configuration - use `docker stack deploy` to deploy to a swarm.

Reply:
yes; that part is specifically for `docker stack deploy`, which I also tested